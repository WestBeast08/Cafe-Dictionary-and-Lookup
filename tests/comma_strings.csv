census_year,block_id,property_id,base_property_id,building_address,clue_small_area,business_address,trading_name,industry_code,industry_description,seating_type,number_of_seats,longitude,latitude
2020,369,103507,103507,Court House Hotel 86-90 Errol Street NORTH MELBOURNE VIC 3051,North Melbourne,Court House Hotel 86-90 Errol Street NORTH MELBOURNE VIC 3051,Court House Hotel,4520,"Pubs, Taverns and Bars",Seats - Indoor,100,144.94996,-37.80288
2020,42,101156,101155,535 Bourke Street MELBOURNE VIC 3000,Melbourne (CBD),"Shop 1B, 535 Bourke Street MELBOURNE VIC 3000",Doughboys Doughnuts,4511,Cafes and Restaurants,Seats - Outdoor,24,144.95781,-37.81637
