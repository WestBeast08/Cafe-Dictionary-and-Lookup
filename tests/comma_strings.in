Court House Hotel
Doughboys Doughnuts
