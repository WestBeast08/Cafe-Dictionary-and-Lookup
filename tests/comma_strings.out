Court House Hotel
--> census_year: 2020 || block_id: 369 || property_id: 103507 || base_property_id: 103507 || building_address: Court House Hotel 86-90 Errol Street NORTH MELBOURNE VIC 3051 || clue_small_area: North Melbourne || business_address: Court House Hotel 86-90 Errol Street NORTH MELBOURNE VIC 3051 || trading_name: Court House Hotel || industry_code: 4520 || industry_description: Pubs, Taverns and Bars || seating_type: Seats - Indoor || number_of_seats: 100 || longitude: 144.94996 || latitude: -37.80288 || 
Doughboys Doughnuts
--> census_year: 2020 || block_id: 42 || property_id: 101156 || base_property_id: 101155 || building_address: 535 Bourke Street MELBOURNE VIC 3000 || clue_small_area: Melbourne (CBD) || business_address: Shop 1B, 535 Bourke Street MELBOURNE VIC 3000 || trading_name: Doughboys Doughnuts || industry_code: 4511 || industry_description: Cafes and Restaurants || seating_type: Seats - Outdoor || number_of_seats: 24 || longitude: 144.95781 || latitude: -37.81637 || 
